src/Lister/SubPageList.php

```diff
@@ -91,13 +91,15 @@ private function renderForTitle( Title $title, array $parameters ) {
 	/**
 	 * @param Title $title
 	 * @param int $limit
+	 * @param string $sortOrder
 	 * @param bool $includeRedirects
 	 *
 	 * @return Page
 	 * @throws LogicException
 	 */
-	private function getPageHierarchy( Title $title, $limit, $includeRedirects ) {
+	private function getPageHierarchy( Title $title, $sortOrder, $limit, $includeRedirects ) {
 		$this->subPageFinder->setLimit( $limit );
+		$this->subPageFinder->setSortOrder( $sortOrder );
```

Comment:
This relies on the constant values matching the user input, which is bad practice